spec/features/admin_sees_payments_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature "Admin sees payments" do
+  background do
+    alice = Fabricate(:user, full_name: "Alice Doe", email: "alice@example.com")
+    Fabricate(:payment, amount: 999, user: alice)
+  end
+
+  scenario "admin can see payments" do
+    sign_in(Fabricate(:admin))
+    visit admin_payments_path
+    expect(page).to have_content("$9.99")
+    expect(page).to have_content("Alice Doe")
+    expect(page).to have_content("alice@example.com")
+  end
+
+  scenario "user cannot see payments" do
+    sign_in(Fabricate(:user))
+    visit admin_payments_path
+    expect(page).not_to have_content("$9.99")
```

Comment:
The inverse of `to have_content` is actually `to have_no_content`. You can read more [here](http://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#has_no_text%3F-instance_method).